Ruby/as1_ruby.rb

```diff
@@ -0,0 +1,97 @@
+#start of Program
+puts "enter the size of the grid"
+$size =  gets.chomp.to_i
+
+#Initializing Array
+$array = Array.new($size){Array.new($size , 0)}
```

Comment:
There should be no space b/w parameter and comma but after the comma, there should be one space before the second parameter.
Array.new($size, 0)